### PR TITLE
16 home image

### PR DESCRIPTION
**Pull request by Cranekick50**

### CODE REVIEW

**Comment by Cranekick50** [comment]
Debugged the background picture, added jquery link, added ingredient search html page, added ingredient search function.

**Review by AshleyLerma — APPROVED** [review]
Looks 🔥

File: assets/script.js [hunk 1]
```diff
@@ -37,3 +37,40 @@ function searchCocktail(cocktailVal) {
     }
   });
 }
+
+
+$(".ingredientSubBtn").click(function(){
+  // event.preventDefault();
+  let value = $(".ingredientInfo").val();
```
Comment (juanlajara) [inline assets/script.js:44]: I would add the trim method/function to the end of the .val

**Review by juanlajara — APPROVED** [review]
Great Work! Please see my comments about adding the trim method I mentioned.

**Review by joshuabogart — APPROVED** [review]

**Review by Frazerhughes — APPROVED** [review]
hey was out of town and traveling home today, sorry for holding things up :(

**Pull request merged**